### PR TITLE
Beaglebone rename

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

**Comment by drewmoseley** [comment]
This is an attempt to handle the beaglebone rename.  I've built locally successfully.  I _think_ I may have the CI stuff updated properly but @GregorioDiStefano @kacf please review carefully.

**Comment by kacf** [comment]
This failed the Beaglebone build actually, but the status update to Github was apparently missing. I have triggered a new one.

**Comment by kacf** [comment]
But certainly this will require changes in mender-qa as well. I think I will just leave this open until I do the next poky/master update, and cherry-pick it then.

File: tests/meta-mender-beaglebone-ci/conf/machine/beaglebone-mender.conf [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 require conf/machine/beaglebone.conf
 
-MACHINEOVERRIDES =. "beaglebone:"
+MACHINEOVERRIDES =. "beaglebone:beaglebone-yocto:"
```
Comment (kacf) [inline tests/meta-mender-beaglebone-ci/conf/machine/beaglebone-mender.conf]: If Yocto has moved away from `beaglebone` we should probably stop referring to it altogether, so I propose removing the override. Do you agree? I'm testing whether this causes any problems in #441.

Comment (drewmoseley) [reply tests/meta-mender-beaglebone-ci/conf/machine/beaglebone-mender.conf]: Fixing the MACHINEOVERRIDES and MENDER_DEVICE_TYPES_COMPATIBLE seem like reasonable changes to make.  I'll take a look in #441

**Pull request closed without merging**